src/System.Security.Cryptography.Xml/tests/EncryptionPropertyTest.cs

```diff
@@ -0,0 +1,128 @@
+            Assert.NotNull(encryptionProperty.PropertyElement);
+            Assert.Equal(doc.DocumentElement, encryptionProperty.PropertyElement);
+        }
+
+        [Fact]
+        public void ConstructorWithXmlElement_Element_Null()
+        {
+            Assert.Throws<ArgumentNullException>(() => new EncryptionProperty(null));
+        }
+
+        [Fact]
+        public void ConstructorWithXmlElement_InvalidXml()
+        {
+            string xml = "<a />";
+            XmlDocument doc = new XmlDocument();
+            doc.LoadXml(xml);
+            Assert.Throws<CryptographicException>(() => new EncryptionProperty(doc.DocumentElement));
+        }
+
+        [Fact]
+        public void PropertyElement_Set_Null()
+        {
+            EncryptionProperty encryptionProperty = new EncryptionProperty();
+            Assert.Throws<ArgumentNullException>(() => encryptionProperty.PropertyElement = null);
+        }
+
+        [Fact]
+        public void PropertyElement_Set_InvalidXml()
+        {
+            string xml = "<a />";
+            XmlDocument doc = new XmlDocument();
+            doc.LoadXml(xml);
+            EncryptionProperty encryptionProperty = new EncryptionProperty();
+            Assert.Throws<CryptographicException>(() => encryptionProperty.PropertyElement = doc.DocumentElement);
+        }
+
+        [Fact]
+        public void PropertyElement_Set_ValidEncryptionPropertyXml()
+        {
+            XmlDocument doc = new XmlDocument();
+            doc.LoadXml(ValidXml);
+            EncryptionProperty encryptionProperty = new EncryptionProperty();
+            encryptionProperty.PropertyElement = doc.DocumentElement;
+            Assert.Equal(doc.DocumentElement, encryptionProperty.PropertyElement);
+        }
+
+        [Fact]
+        public void LoadXml_Null()
+        {
+            EncryptionProperty encryptionProperty = new EncryptionProperty();
+            Assert.Throws<ArgumentNullException>(() => encryptionProperty.LoadXml(null));
+        }
+
+        [Fact]
+        public void LoadXml_InvalidXml()
+        {
+            string xml = "<a />";
+            XmlDocument doc = new XmlDocument();
+            doc.LoadXml(xml);
+            EncryptionProperty encryptionProperty = new EncryptionProperty();
+            Assert.Throws<CryptographicException>(() => encryptionProperty.LoadXml(doc.DocumentElement));
+        }
+
+        [Fact]
+        public void LoadXml_ValidEncryptionPropertyXml()
+        {
+            XmlDocument doc = new XmlDocument();
+            doc.LoadXml(ValidXml);
+            EncryptionProperty encryptionProperty = new EncryptionProperty();
+            encryptionProperty.LoadXml(doc.DocumentElement);
+            Assert.Equal(doc.DocumentElement, encryptionProperty.PropertyElement);
+            Assert.Equal("ID_val", encryptionProperty.Id);
+            Assert.Equal("Target_val", encryptionProperty.Target);
```

Comment:
These strings probably make more sense as consts up near the XML.  I was confused as to if they were from data (they were) or a spec, or were defaults, or what.